cmd/check.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/odpf/salt/printer"
+	"github.com/odpf/salt/term"
+	stencilv1beta1 "github.com/odpf/stencil/proto/odpf/stencil/v1beta1"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc/status"
+)
+
+func checkSchemaCmd() *cobra.Command {
+	var host, comp, file, namespaceID string
+	var req stencilv1beta1.CheckCompatibilityRequest
+
+	cmd := &cobra.Command{
+		Use:   "check <id>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Check schema compatibility",
+		Long: heredoc.Doc(`
+			Check schema compatibility of a local schema
+			against a remote schema(against) on stencil server.`),
+		Example: heredoc.Doc(`
+			$ stencil schema check <id> -n odpf -c COMPATIBILITY_BACKWARD -F ./booking.desc
+	    `),
```

Comment:
This looks like wrong formatting but this is not appear in my local when I clone it locally.

Reply:
Yes, somehow only GitHub is showing this.

Reply:
@mabdh fixed. :) space vs tabs :D